honeybee_radiance/modifier/material/antimatter.py

```diff
@@ -9,9 +9,11 @@
 class Antimatter(Material):
     """Radiance Antimatter Material.
 
-    Antimatter is a material that can "subtract" volumes from other volumes. A ray
+    Antimatter is a material that can \'subtract\' volumes from other volumes. A ray
```

Comment:
This is a good thing to be aware of. Let's make sure that we document this in the wiki.

Also, I realized that this class lacks `Args` and `Properties`.

Reply:
Indeed, I am making notes as I go through them, this one was sneaky!

Reply:
In regards to the classes without Arguments/Properties, I actually had Args/Properties in all of them and deleted them after. Bad judgement!
I had the feeling this classes weren't ready to be used, sort of a placeholder, and that those properties will probably change.  Clearly they may still be declared by users even as is.

Reply:
I guess I realize now I forgot to delete the Args/Properties from some classes, that's why you saw the inconsistency. Glad you found them useful!

Reply:
You are right on all counts. These classes are technically placeholders that should hopefully get fleshed out into something more detailed in the future. But they are still usable in their current state so it's helpful to document the Args and Properties for the time being.